src/components/ToDo/LastRow.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from 'react'
+import Todo from '../../pages/FrontendMentor/Todo'
+import { ItemsLeft, MiddleTBox, Rectangle, TextBox, Clear } from './styles/LastRow.styled'
+type todo = {
+  key: string
+  task: string
+  completed: boolean
+  visible: boolean
+}
+
+type Props = {
+  todos: todo[]
+  setTodos: any
+  isDarkMode: boolean
+}
+
+const LastRow = (props: Props) => {
+  const clearCompleted = () => {
+    const filtered = props.todos.filter((task) => {
```

Comment:
consider using more meaningful names, like pendingTasks or uncompletedTasks instead of filtered